asyncapi/v2/testdata/example-kafka.yaml

```diff
@@ -0,0 +1,163 @@
+asyncapi: '2.0.0'
+info:
+  title: Streetlights Kafka API
+  version: '1.0.0'
+  description: |
+    The Smartylighting Streetlights API allows you to remotely manage the city lights.
+    ### Check out its awesome features:
+    * Turn a specific streetlight on/off 🌃
+    * Dim a specific streetlight 😎
+    * Receive real-time information about environmental lighting conditions 📈
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+
+servers:
+  test:
+    url: localhost:9092
+    protocol: kafka-secure
+    description: Test broker
+    security:
+      - saslScram: []
+    x-eventgateway-listener: proxy:28002
```

Comment:
Is there a way we can avoid having to use this extension?

Reply:
Yes, we can do few things here:

1. Open ports automatically by using a default range. Like, for every broker the proxy will open a port starting by 28002 (meaning other brokers will have 28003, 28004, etc)
2. Open ports randomly. Then the opened ports list should be published somewhere so clients should discover it.
3. Nothing else comes to my mind now but most probably there are?

Reply:
Why can't we use the same ports as Kafka? I'm not sure I understand 🤔

Reply:
Because different brokers can use the same port number. For example: `test.mykafkacluster.org:8092`, `test2.mykafkacluster.org:8092`, `test3.mykafkacluster.org:8092`, `test4.mykafkacluster.org:8092`

Reply:
You could of course run one instance of the proxy per broker, but it is not mandatory at all (even though in most cases, recommended).

Reply:
Yeah, I was assuming one gateway instance per broker, otherwise, we'd be encouraging a bad pattern where people will complain about the performance of the gateway when in reality it's that they're using one for multiple brokers 😅

Reply:
Ok, WDYT about providing both options? 

1. Configure to proxy all brokers (I see this very useful for small environments and/or for testing purposes) 
2. Tell the Proxy via env var which AsyncAPI server should use on that particular gateway instance, so the server port can be grabbed in order to configure the Kafka proxy.

I would also still keep the possibility to configure everything manually from env vars as it is now (in fact it is needed if you wanna run proxy and brokers in different networks without having to change DNS, like in local Docker).

Reply:
Yeah, sounds sensible to me 👍

Reply:
Would it make sense to merge this PR and do the changes in another one? Everything is under development so we can do this safely.